spec/jobs/send_message_job_spec.rb

```diff
@@ -88,6 +88,22 @@
             described_class.perform_now recipient.id
           end
         end
+        context "when the recipient's name is same case with spaces" do
```

Comment:
I like the change that capitalizes each word of a multi-word `first_name`, e.g., "MARY LOU" => "Mary Lou". 

However, I worry about the 2nd change with respect to the Roman numerals. It seems like a weird edge case to me:
1. In my experience, Roman numerals, "Jr", etc. are with the last name, not the first name - we are really dealing with bad data here.
2. This is customization of a customization - I would find it extremely hard to come up with a generic version of this solution in the spirit of https://www.pivotaltracker.com/story/show/184206508.

Reply:
I removed the roman numerals, leaving it with capitalization for spaces and hyphens